src/Button/BorderlessButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {
+  FunctionComponent, ReactElement, useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { darken } from 'polished';
+import { BaseTheme } from '../Theme';
+import { StyledButton } from './StyledButton';
+import { ButtonProps } from './ButtonProps';
+
+const StyledPrimaryButton = styled(StyledButton)`
+  background: ${({ theme }): string => (theme.color.background.info.light)};
+  color: ${({ theme }): string => (theme.color.text.light)};
```

Comment:
If we do go with a transparent background, it would probably make sense for `color` to be a prop, so you can adjust for contrast.

Reply:
Fixed through 769e9fd